Automattic/wp-calypso
# Trail Map: Add experiment based features

Related to #

## Proposed Changes

* Adds all changed features for trail map behind a feature flag.

## Testing Instructions



* Go to `/start/plans?flags=onboarding/trail-map-feature-grid`
* Make sure the new feature copy is visible for related features
     * Link - [Doc](https://docs.google.com/spreadsheets/d/1Bz8W76rR-_q_osFIhx9Lsc5n_BJN5FIQ/edit#gid=1929114643)

## Pre-merge Checklist
